docker-compose.yml

```diff
@@ -328,6 +329,7 @@ services:
       apollo:
         condition: service_healthy
     image: docker.elastic.co/kibana/kibana:8.12.2
+    #container_name: iris
```

Comment:
added container name as well but commented so that it doesn't break anything for twang